tests/vite-plugin-react-pages.ts

```diff
@@ -0,0 +1,16 @@
+import { runInRepo } from '../utils'
+import { RunOptions } from '../types'
+
+export async function test(options: RunOptions) {
+	await runInRepo({
+		...options,
+		repo: 'vitejs/vite-plugin-react-pages',
+		overrides: {
+			'@vitejs/plugin-react': true,
+		},
+		branch: options.viteMajor === 4 ? 'dev/v4' : 'main',
```

Comment:
is this still needed or is vite4 support now on main?

Reply:
It is still needed because `dev/v4` has not been merged into the main branch

Reply:
@cijiugechu I just merge `dev/v4` into `main`. We can go with the main branch now.